app/components/show/controls_component.rb

```diff
@@ -157,10 +157,6 @@ def in_accessioning?
       doc['processing_status_text_ssi']&.include?('In accessioning')
     end
 
-    def workflow_errors?
-      doc[SolrDocument::FIELD_WORKFLOW_ERRORS].present?
-    end
-
```

Comment:
added for this button earlier, but we don't need it now that we have incorporated ocrWF into the existing check for assembling